src/Select/Select.tsx

```diff
@@ -1,107 +1,52 @@
+   */
+  label: string;
+  children: ReactElement<'option'>[];
+  /**
+   * Gestisce il cambio del valore selezionato
+   * @param selectedValue
+   * @returns
+   */
+  handleChange: (selectedValue: string) => void;
 }
 
-const DropdownIndicator = <
-  OptionType extends OptionTypeBase,
-  IsMulti extends boolean,
-  GroupType extends GroupTypeBase<OptionType> = GroupTypeBase<OptionType>
->(
-  props: IndicatorProps<OptionType, IsMulti, GroupType>
-) => {
-  return (
-    <components.DropdownIndicator {...props}>
-      <Icon icon='it-arrow-down-triangle' aria-hidden />
-    </components.DropdownIndicator>
-  );
-};
-
-const Option = <
-  OptionType extends OptionTypeBase,
-  IsMulti extends boolean,
-  GroupType extends GroupTypeBase<OptionType> = GroupTypeBase<OptionType>
->(
-  props: OptionProps<OptionType, IsMulti, GroupType>
-) => {
-  return (
-    <div className='select-pill text-primary'>
-      <components.Option {...props} />
-    </div>
-  );
-};
-
-const CustomClearText: FC<{ clearText: string }> = ({ clearText }) => (
-  <>{clearText}</>
-);
-
-export const Select = <
-  OptionType extends OptionTypeBase = { label: string; value: string },
-  IsMulti extends boolean = false,
-  GroupType extends GroupTypeBase<OptionType> = GroupTypeBase<OptionType>
->({
-  clearText = 'Annulla',
-  icon = false,
-  ...props
-}: SelectProps<OptionType, IsMulti, GroupType>): ReactElement<
-  SelectProps<OptionType, IsMulti, GroupType>
-> => {
-  // @TODO: move it outside when migrating to hooks
-  const ClearIndicator = <
-    OptionType extends OptionTypeBase,
-    IsMulti extends boolean,
-    GroupType extends GroupTypeBase<OptionType> = GroupTypeBase<OptionType>
-  >(
-    clearIndicatorProps: IndicatorProps<OptionType, IsMulti, GroupType>
-  ) => {
-    const {
-      children = <CustomClearText clearText={clearText} />,
-      innerProps: { ref, ...restInnerProps }
-    } = clearIndicatorProps;
-    return (
-      <div className='select-pill text-primary' {...restInnerProps} ref={ref}>
-        <div style={{ padding: '0px 5px' }}>{children}</div>
-      </div>
-    );
+export const Select = ({
+  id = 'select',
+  disabled = false,
+  label,
+  children,
+  handleChange
+}: SelectProps) => {
+  const _handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    console.log(e);
+    handleChange(e.target.value);
   };
-
-  const selectComponents = {
-    ...props.components,
-    Option,
-    DropdownIndicator,
-    IndicatorSeparator: null
-  };
-
-  if (props.isClearable) {
-    selectComponents.ClearIndicator = ClearIndicator;
-  }
   return (
-    <SelectBase {...props} components={selectComponents} styles={styles} />
+    <div className='select-wrapper'>
+      <label htmlFor={id + '-select'}>{label}</label>
+      <Input
+        id={id + '-select'}
+        name='select'
+        type='select'
+        disabled={disabled}
+        onChange={_handleChange}
+      >
+        <option value=''>Scegli un'opzione</option>
```

Comment:
I would leave this to the consumer to define it

Reply:
You mean the option with the empty value? You're right, there's no reason to enforce it